### DISCUSSION TITLE
[Config Support]: Ratio of Motion Size to Object Detection?

### DISCUSSION THREAD

**Discussion by mrmees** [blakeblackshear/frigate#12644] [Config Support]
### Describe the problem you are having

Is there a way to require a minimum motion area size on a per object basis?  

Have an area with parked vehicles and cats are triggering a motion alert for them.  It seems like motion - contour_area might work, but at the expense of detecting anything relatively cat sized.  

It's not critical, but I can see a situation where I'd like to track the cats too.  

![Untitled](https://github.com/user-attachments/assets/43b6aa45-f11e-4d28-b4ca-2b80a2bef0cd)

### Version

0.14.0-a4eb435

### Frigate config file

```yaml
N/A
```

### Relevant log output

```shell
N/A
```

### Operating system

Proxmox

### Install method

Docker Compose

### Object Detector

Coral

**Comment by NickM-27** [comment]
You can't change "motion size" per object, the whole point of motion detection is looking for objects. So you can't use information from the future to change a decision in the present. 

The only time an alert occurs is if the objects box moves enough to not be considered stationary anymore. This seems like it is unrelated to motion detection.
